parkings/models/zone.py

```diff
@@ -23,3 +23,9 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+    def get_code(self):
+        try:
+            return int(self.code)
+        except ValueError:
+            return self.code
```

Comment:
This function could probably have a more descriptive name, and maybe it should be a property, since it's really light computation, not really "getting" or "fetching" anything from DB or anywhere. Also, I would use isdigit to avoid the try-catch. How about something like this?
```python
@property
def casted_code(self):
    return int(self.code) if self.code.isdigit() else self.code
```